uui-components/src/navigation/Anchor.module.scss

```diff
@@ -2,9 +2,14 @@
     text-decoration: none;
     cursor: pointer;
     outline: none;
-  
+
     &:global(.uui-disabled) {
         cursor: default;
     }
-  
+
+    &:global(.-clickable):focus-visible {
```

Comment:
Why we don't use here new `focus-visible-effect` mixin?

Reply:
Because this mixin is placed in `uui` package. 
I moved focus styling for `Anchor` in `uui` package and used this mixin.